Dockerfile-ubuntu-22.04

```diff
@@ -16,7 +16,10 @@ RUN apt-get install -y curl gnupg && \
 curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg && \
 mv bazel.gpg /etc/apt/trusted.gpg.d/ && \
 echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list && \
-apt-get update -y && apt-get install -y bazel
+apt-get update -y && apt-get install -y bazel-6.4.0
```

Comment:
curious what the default version was?

https://bazel.build/install/ubuntu recommends to install bazelisk, so maybe we should do that?

Reply:
The default version from the apt install is 7.6.0. Sure, I will test the Bazelisk and then update this PR